src/test/java/internship/framework/pages/PersonalDetailsPage.java

```diff
@@ -0,0 +1,191 @@
+        return this;
+    }
+
+    public PersonalDetailsPage fillWithValueCityTown() {
+        ProjectLogger.info("Fill city/town element with value.");
+        ElementUtils.fillInWithValue(cityTownElement, TestData.YOUR_DETAILS_CITY_TOWN);
+        return this;
+    }
+
+    public PersonalDetailsPage selectCountry() {
+        ProjectLogger.info("Select country.");
+        Select select = new Select(countryElement);
+        select.selectByVisibleText("Romania");
+        return this;
+    }
+
+    public PersonalDetailsPage fillWithValueTelephone() {
+        ProjectLogger.info("Fill telephone element with value.");
+        ElementUtils.fillInWithValue(telephoneElement, TestData.YOUR_DETAILS_TELEPHONE);
+        return this;
+    }
+
+    public boolean isValidateChangesDisplayed() {
+        ProjectLogger.info("Check if validate changes element is displayed.");
+        return ElementUtils.doesAppear(validateChangesElement);
+    }
+
+    public void clickSaveChangesButton() {
+        ProjectLogger.info("Save changes made.");
+        ElementUtils.clickOn(saveChangesElement);
+    }
+
+    public PersonalDetailsPage selectAllContactPreferences() {
+        ProjectLogger.info("We check all contact preferences.");
+        ElementUtils.clickOn(firstCheckBoxElement);
+        ElementUtils.clickOn(secondCheckBoxElement);
+        ElementUtils.clickOn(thirdCheckBoxElement);
+        ElementUtils.clickOn(fourthCheckBoxElement);
+        return this;
+    }
+
+    public PersonalDetailsPage navigateSubscriptionDetailsTab() {
+        ProjectLogger.info("Navigate to Subscription Details page.");
+        ElementUtils.clickOn(subscriptionDetailsButton);
+        return this;
+    }
+
+    public void navigatePersonalDetailsTab() {
+        ProjectLogger.info("Navigate to Personal Details page.");
+        ElementUtils.clickOn(personalDetailsButton);
+    }
+
+    private boolean isFirstCheckBoxSelected() {
+        ProjectLogger.info("Check if first contact checkbox is selected.");
+        Driver.get().manage().timeouts().implicitlyWait(5, TimeUnit.SECONDS);
+        return Boolean.parseBoolean(checkFirstCheckboxElement.getAttribute("checked"));
+    }
+
+    private boolean isSecondCheckBoxSelected() {
+        ProjectLogger.info("Check if second contact checkbox is selected.");
+        return Boolean.parseBoolean(checkSecondCheckboxElement.getAttribute("checked"));
+    }
+
+    private boolean isThirdCheckBoxSelected() {
+        ProjectLogger.info("Check if third contact checkbox is selected.");
+        return Boolean.parseBoolean(checkThirdCheckboxElement.getAttribute("checked"));
+    }
+
+    private boolean isFourthCheckBoxSelected() {
+        ProjectLogger.info("Check if fourth contact checkbox is selected.");
+        return Boolean.parseBoolean(checkFourthCheckboxElement.getAttribute("checked"));
+    }
+
+    public boolean areALLCheckboxesSelected() {
+        return isFirstCheckBoxSelected() & isSecondCheckBoxSelected()
```

Comment:
Why did you use & instead of && ? Please read the following article : https://www.dummies.com/programming/java/and-operators-and-in-java/

Reply:
My initial code include &&, but I got an advice from IntelliJ to simplify code this way. I've read the article and understand that is beneficial to use && performance-wise.